Tests/UnitTests/Core/EventData/MultiTrajectoryTests.cpp

```diff
@@ -243,31 +248,30 @@ BOOST_AUTO_TEST_CASE(ApplyWithAbort) {
 BOOST_AUTO_TEST_CASE(BitmaskOperators) {
   using PM = TrackStatePropMask;
 
-  auto bs1 = PM::Uncalibrated;
```

Comment:
Just curiosity. Is that realted to backend cahnge or it is a collateral change?

Reply:
As per the other comment: it's kind of collateral change, but I thought it made sense to changed it with this PR, since I'm touching the columns anyway.